client/src/components/CreateMeetup/CreateMeetup.js

```diff
@@ -1,7 +1,246 @@
+  const [desc, setDesc] = useState("");
+  const [tags, setTags] = useState([]);
+
+  // TODO: load options from the server instead
+  const tagOptions = [
+    "basketball",
+    "ping pong",
+    "badminton",
+    "movie",
+    "cooking",
+  ];
+
+  const renderNameInput = () => {
+    return (
+      <div className={classes.formSection}>
+        <TextField
+          required
+          variant="outlined"
+          id="meetup-name"
+          label="Meetup Name"
+          value={name}
+          onChange={(event) => setName(event.target.value)}
+          className={classes.formInput}
+        />
+      </div>
+    );
+  };
+
+  const renderDateInput = () => {
+    return (
+      <div className={classes.formSection}>
+        <MuiPickersUtilsProvider utils={DayUtils}>
+          <KeyboardDatePicker
+            variant="inline"
+            id="date-picker-inline"
+            inputVariant="outlined"
+            format="MM/DD/YYYY"
+            label="Meetup Date"
+            value={date}
+            onChange={(newDate) => setDate(newDate)}
+            className={classes.formInput}
+            style={{
+              marginRight: 20,
+            }}
+          />
+          <KeyboardTimePicker
+            variant="inline"
+            inputVariant="outlined"
+            id="time-picker"
+            label="Meetup Time"
+            value={date}
+            onChange={(newDate) => setDate(newDate)}
+            className={classes.formInput}
+          />
+        </MuiPickersUtilsProvider>
+      </div>
+    );
+  };
+
+  const renderMeetupLocation = () => {
+    return (
+      <Box display="flex" flexWrap="wrap" className={classes.formSection}>
+        <FormControl variant="outlined" className={classes.formInput}>
+          <InputLabel id="select-meetup-type-label">Meetup Type</InputLabel>
+          <Select
+            label="Meetup Type"
+            id="select-meetup-type"
+            labelID="select-meetup-type-label"
+            value={meetupType}
+            onChange={(event) => setMeetupType(event.target.value)}
+            className={classes.formInput}
+          >
+            <MenuItem value="In person">In person</MenuItem>
+            <MenuItem value="Remote">Remote</MenuItem>
+          </Select>
+        </FormControl>
+        {meetupType === "Remote" ? ( // TODO: add location support for in-person events
+          <TextField
+            required
+            id="meetup-location"
+            label="Meetup Location"
+            variant="outlined"
+            value={meetupLocation}
+            onChange={(event) => setMeetupLocation(event.target.value)}
+            className={classes.formInput}
+            style={{
+              marginLeft: 20,
+            }}
+          />
+        ) : null}
+      </Box>
+    );
+  };
+
+  const renderGroupSizeInput = () => {
+    return (
+      <Box display="flex" alignItems="baseline" className={classes.formSection}>
+        <Typography>Group Size: </Typography>
```

Comment:
Consider doing something like this:

https://material-ui.com/components/slider/#customized-sliders, under "Airbnb". OKay to do as a follow-up, or even not at all.

Reply:
This is a cool idea, I'd be down to experiment with it as a follow-up thing later on!